source/resources/index.html

```diff
@@ -112,10 +112,12 @@ <h3>Social Media</h3>
           </div>
           <div class="col-sm-6">
             <h3>Practice Online</h3>
-          	<ul class="shy-list">
-              <li><a href="http://exercism.io/languages/perl6">Perl&nbsp;6 track on exercism.io</a></li>
-    	        <li><a href="https://glot.io/new/perl6">Online Perl&nbsp;6 REPL</a></li>
-            </ul>
+              <ul class="shy-list">
+                  <li><a href="http://exercism.io/languages/perl6">Perl&nbsp;6 track on exercism.io</a></li>
+       	          <li><a href="https://glot.io/new/perl6">Online Perl&nbsp;6 REPL</a></li>
+                  <li><a href="https://tio.run/#perl6">Online Perl&nbsp;6 REPL</a></li>
+                  <li><a href="https://ideone.com/">Online Perl&nbsp;6 REPL</a></li>
```

Comment:
Isn't it weird to have three repeated links that say "Online Perl 6 REPL"? Could've used the name of the REPL or something.